src/components/views/createplayer/CreatePlayer.tsx

```diff
@@ -39,36 +38,31 @@ const CreatePlayer = () => {
 
   return (
     <div className="div-box">
-      <BaseContainer>
-        <h1 className="font-title">Welcome to "Screw your neighbour"</h1>
-      </BaseContainer>
-      <br />
-      <BaseContainer>
-        <h4>
-          Register by entering your name in the field below and pressing "Take
-          me to the Lobby"
-        </h4>
-        <form onSubmit={submit}>
-          <Box className="Box">
-            <TextField
-              helperText="Please enter your player name"
-              id="demo-helper-text-aligned"
-              label="player name"
-              value={name}
-              onChange={changeName}
-            />
-            <p></p>
-            <Button
-              type={"submit"}
-              disabled={loading}
-              variant="contained"
-              endIcon={<SendIcon />}
-            >
-              TAKE ME TO THE LOBBY
-            </Button>
-          </Box>
-        </form>
-      </BaseContainer>
+      <div className="div-container">
+        <BaseContainer>
+          <h1 className="font-title">Welcome to "Screw your neighbour"</h1>
+        </BaseContainer>
+        <BaseContainer>
+          <h4>
+            Register by entering your name and pressing "Take me to the Lobby"
+          </h4>
+          <form onSubmit={submit}>
+            <Box className="Box">
+              <TextField
+                helperText="Please enter your player name"
+                id="demo-helper-text-aligned"
+                label="player name"
+                value={name}
+                onChange={changeName}
+              />
+              <p></p>
+              <button className="button" disabled={loading} onClick={submit}>
```

Comment:
ich würde dafür plädieren, eine Button komponente zu machen, und nicht ein globales css zu buttons.scss zu verwenden.
(Und vielleicht können wir ja doch noch bisschen material verwenden, alles müssen wir ja nicht selber machen).
Muss nicht unbedingt in diesem PR passieren, aber ich würde darauf hinarbeiten.